zlib is broken

```
In file included from /opt/nxdk/lib/zlib/zlib/zlib.h:34:
/opt/nxdk/lib/zlib/zlib/zconf.h:444:14: fatal error: 'sys/types.h' file not found
#    include <sys/types.h>      /* for off_t */
             ^~~~~~~~~~~~~
1 error generated.
```

--- a/lib/pkgconfig/zlib.pc
+++ b/lib/pkgconfig/zlib.pc
@@ -4,4 +4,4 @@ Version: 1.2.11
 
 Requires:
 Libs: ${NXDK_DIR}/lib/libzlib.lib
-Cflags: -I${NXDK_DIR}/lib/zlib/zlib
+Cflags: -I${NXDK_DIR}/lib/zlib/zlib -DZ_SOLO

